main/main.cpp

```diff
@@ -545,6 +548,10 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			}
 		} else if (I->get() == "-u" || I->get() == "--upwards") { // scan folders upwards
 			upwards = true;
+		} else if (I->get() == "--build-solutions") { // Build the scripting solution such C#
```

Comment:
I guess this one should also be enclosed in `#ifdef TOOLS_ENABLED`.

Reply:
I'd be happy to add it. The main reason I didn't is because there aren't any other command line flags, including for --editor, --project-manager, etc, that have the TOOLS_ENABLED compiler flag. Up to you :)

Reply:
Actually if you rebase, they should have by now :)

Reply:
Cool, thanks for letting me know. They're now moved into the compiler flag :)